### PR TITLE
selenium-chrome-driver: upgrade to 2.37

### PR DESCRIPTION
**Pull request by vtintillier**

### CODE REVIEW

**Comment by vtintillier** [comment]
Hello @agabrys . We need 2.37 as 2.36 is blocked as Trojan (!) by McAfee...

Thanks

File: manual/selenium-chrome-driver/x86/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -5,8 +5,8 @@ $driverPath = "$seleniumDir\chromedriver.exe"
 
 $packageArgs = @{
   packageName   = 'selenium-chrome-driver'
-  url           = 'https://chromedriver.storage.googleapis.com/2.36/chromedriver_win32.zip'
-  checksum      = '17d485ab082b5c07d4d0d9894616a94fac5cef8ceaaf840ced07f617d274d8bfa4538610a2eea6ab3ba1e024cd8f8c28b600fb5b49b4206f1790c039f5a34f89'
+  url           = 'https://chromedriver.storage.googleapis.com/2.37/chromedriver_win32.zip'
+  checksum      = '947ed273fa498ccb5e14dba511d2b2ab0e40fa6ff273af631471402c8aadb01cb921d27c00e9ee87db3744f257745525067113cedf3de9c7c71a2d5a57556fd6'
```
Comment (agabrys) [inline manual/selenium-chrome-driver/x86/tools/chocolateyInstall.ps1:9]: A correct checksum of the file is:
`189f1942161f8696ac79b26eb20712c6c797e58a4c83081584e182eda6213ff325bc385675c733207f655cd858d5b737ce86c7574c258ab2700955438979c003`

Maybe they overwrote `chromedriver_win32.zip` file after you had prepared the PR.

**Review by agabrys — CHANGES_REQUESTED** [review]
The checksum is incorrect.

**Comment by agabrys** [comment]
I updated the package, see: 265c890a4a65c10856feff5da8daaff3e5ba438a

**Comment by vtintillier** [comment]
Thank you @agabrys . I now realize I checksummed the exe file, not the zip 😞

**Pull request closed without merging**